src/include/miopen/ctc.hpp

```diff
@@ -56,7 +56,7 @@ struct CTCLossDescriptor : miopenCTCLossDescriptor
                                    const int* inputLengths,
                                    miopenCTCLossAlgo_t algo) const;
 
-    void CTCLoss(const Handle& handle,
+    void CTCLoss(Handle& handle,
```

Comment:
CTCLoss calls SoftmaxForward which subsequently calls SolverContainers::ExecutePrimitive which accept non const Handle&.
It leads to CTCLoss should accept non const Handle&. Otherwise we need to use const-cast hack.

Reply:
@Vsevolod1983 I see, thanks.

@DrizztDoUrden What about https://github.com/ROCm/MIOpen/pull/1276#discussion_r1455119631?